src/Init/Data/Char/Lemmas.lean

```diff
@@ -31,6 +31,11 @@ theorem utf8Size_eq (c : Char) : c.utf8Size = 1 ∨ c.utf8Size = 2 ∨ c.utf8Siz
   rw [Char.ofNat, dif_pos]
   rfl
 
+theorem Char.ext : {a b : Char} → a.val = b.val → a = b
```

Comment:
```suggestion
@[ext] theorem Char.ext : {a b : Char} → a.val = b.val → a = b
```
We should preserve the `ext` attribute here. (This enables use of the `ext` tactic to automatically apply any such lemmas.)